src/components/Widgets/Widget.tsx

```
@@ -60,9 +54,18 @@ export function Widget({
   );
 
   useEffect(() => {
-    router.prefetch('/', { kind: PrefetchKind.FULL });
-    router.prefetch('/gas', { kind: PrefetchKind.FULL });
-  }, [router]);
+    const routes = [AppPaths.Main, AppPaths.Gas].filter(
+      (route) => route !== pathname,
+    );
+
+    const id = requestIdleCallback(() => {
+      routes.forEach((route) =>
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)